packages/plugin-auto-tracking-browser/src/helpers.ts

```diff
@@ -51,3 +51,45 @@ export const getText = (element: Element): string => {
   }
   return text;
 };
+
+export const isPageUrlAllowed = (url: string, pageUrlAllowlist: string[] | undefined) => {
+  if (!pageUrlAllowlist || !pageUrlAllowlist.length) {
```

Comment:
No allowlist means, we white list any url?

Reply:
Thanks for reviewing! Yes, if it is not provided, then we don't filter on this.